validphys2/src/validphys/tests/test_pseudodata.py

```diff
@@ -70,8 +71,10 @@ def test_no_savepseudodata():
 
 
 def test_read_matches_recreate():
+    
+
     reads = API.read_fit_pseudodata(fit=PSEUDODATA_FIT)
-    recreates = API.recreate_fit_pseudodata(fit=PSEUDODATA_FIT)
+    recreates = API.recreate_fit_pseudodata(fit=PSEUDODATA_FIT,)
```

Comment:
I think this file you can revert to its original state (so that it obvious there is no changes)